app/models/activity.rb

```diff
@@ -1,3 +1,7 @@
 class Activity < ApplicationRecord
   belongs_to :user
+
+  validates :title, :description, :price, :level, :location, :group_size, presence: true
+  validates :price, :group_size, numericality: { only_integer: true }
+  validates :level, inclusion: { in: ['beginner', 'intermediate', 'advanced'] }
```

Comment:
Here the level field can only have 'beginner, intermediate and advance' do we need to be more broader?

Reply:
Need to change 'advanced' to 'advance'

Reply:
Yes, I agree with these 3 levels.